docs/application/workflow.md

```diff
@@ -0,0 +1,75 @@
+---
+sidebar_position: 4
+---
+
+# Workflows
+
+An application workflow is a set of tasks that are executed in a specific order. The tasks can be executed in parallel or sequentially.  The workflow can be used to automate the deployment of applications, such as the deployment of a web application requires the deployment of multiple services in a specific order.
```

Comment:
Do we have the concept of `application workflow`?

Reply:
updated